docs/webhook-config.md

```diff
@@ -152,13 +156,15 @@ Possible parameters are:
 * `enter_tag`
 
 ### Webhooksell
-
+# TODO-lev add support for long and short?
```

Comment:
Long and short support is there by providing "is_short" ? or am i missing something?